src/bitbucket/BitbucketClient.ts

```diff
@@ -95,11 +95,11 @@ export class BitbucketClient {
   }
 
   async stopLandBuild(buildId: number, lockId?: Date) {
-    return await this.pipelines.stopLandBuild(buildId, lockId);
+    return this.pipelines.stopLandBuild(buildId, lockId);
```

Comment:
oh I don;t remember if eslint is setup In Landkid but it would have been captured - nice